src/svelte/src/utilities/display.ts

```diff
@@ -0,0 +1,260 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const binary_regex = /^[0-1]*$/
+const decimal_regex = /^[0-9]*$/
+const octal_regex = /^[0-7]*$/
+const hex_regex = /^[0-9a-fA-F]*$/
+// const base64_regex =
+//   /^([0-9a-zA-Z+/]{4})*(([0-9a-zA-Z+/]{2}==)|([0-9a-zA-Z+/]{3}=))?$/
```

Comment:
Remove commented code.

Reply:
BASE64 provides a convenient way to insert binary data/files into a session, in which case we would want to validate that data with this regex.